movieApp.js

```diff
@@ -0,0 +1,423 @@
+
+// const img = document.createElement("img");
+// const readMoreBtn = document.createElement("button");
+
+for (let movie of movies) {
+  //Creat a cotainer to store the Image and readMore Button
+
+  const movieContainer = document.createElement("div");
+  movieContainer.className = "movie-container";
+
+  //
+  //  Create an img element and set its src attribute
+  const img = document.createElement("img");
+  img.width = 200;
+  img.src = movie.poster_url;
+
+  // Create a button element
+  const readMoreBtn = document.createElement("button");
+  readMoreBtn.textContent = "Read More";
+  readMoreBtn.className = "accordion";
+  readMoreBtn.style.backgroundColor = "teal";
+  readMoreBtn.style.marginTop = "10px";
+
+  // starRating
+  const starRating = document.createElement("div");
+  starRating.className = "star-rating";
+  const starOne = document.createElement("i");
+  starOne.className = "rating__star far fa-star";
+
+  const starTwo = document.createElement("i");
+  starTwo.className = "rating__star far fa-star";
+
+  const starThree = document.createElement("i");
+  starThree.className = "rating__star far fa-star";
+
+  const starFour = document.createElement("i");
+  starFour.className = "rating__star far fa-star";
+  const starFive = document.createElement("i");
+  starFive.className = "rating__star far fa-star";
+
+  // Append the img and button to the Movie container
+  starRating.appendChild(starOne);
+  starRating.appendChild(starTwo);
+  starRating.appendChild(starThree);
+  starRating.appendChild(starFour);
+  starRating.appendChild(starFive);
+
+  movieContainer.appendChild(img);
+  movieContainer.appendChild(readMoreBtn);
+  movieContainer.appendChild(starRating);
+  // movie container to now streaming
+  nowStreaming.appendChild(movieContainer);
+
+  // Create element to show details about the movie
+  const movieDetails = document.createElement("div");
+  movieDetails.className = "movie-details";
+  movieDetails.style.display = "none";
+  movieDetails.style.color = "white";
+  movieDetails.innerHTML = `
+           <p><strong>Title:</strong> ${movie.title}</p>
+           <p><strong>Description:</strong> ${movie.description}</p>
+           <p><strong>Release Year:</strong> ${movie.movie_year}</p>
+           <p><strong>Director:</strong> ${movie.director}</p>
+           <p><strong>Actors:</strong> ${movie.actors.join(", ")}</p>
+       `;
+  nowStreaming.appendChild(movieDetails);
+
+  // Adding the comment Section
+  const commentSection = document.createElement("div");
+  commentSection.innerHTML = "Comments: ";
+  commentSection.style.fontWeight = "bold";
+  commentSection.className = "comment-section";
+  const commentInput = document.createElement("input");
+  commentInput.type = "text";
+  commentInput.placeholder = "Add a Comment";
+  commentInput.style.color = "white";
+  const commentButton = document.createElement("button");
+  commentButton.className = "comment-button";
+  commentButton.innerHTML = "Submit";
```

Comment:
Small UX suggestion that when you Submit a comment it should set the input field to `""` so you can easily type a new comment!

Reply:
thanks for suggestion, it looks much cleaner and nicer now